pages/about.html

```diff
@@ -4,9 +4,64 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="/styles/style.css" rel="stylesheet">
```

Comment:
Careful with your file paths! The `/styles.css/style.css` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `../styles.css/style.css`. The two dots at the beginning signal to start searching in the parent directory of the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This issue is also happening with the images. It is causing the site to not display the CSS styles and the images do not display.